### PR TITLE
Restaurant dashboard

### PR DESCRIPTION
**Pull request by MikaelFeher**
PT Story: https://www.pivotaltracker.com/story/show/128562003

Changes proposed in this pull request:
- Adds dashboard page on which there are buttons for various
  restaurant management options

What I have learned working on this feature:
- How to add links that look like buttons
- Re-affirmed the notion of how important it is to have a workflow that 
  avoids incomplete features due to unfinished features being interdependent.

Screenshots:
screen shot 2016-08-27 at 22 27 52 2

Ready for review!

### CODE REVIEW

File: app/models/menu.rb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 class Menu < ApplicationRecord
   validates_presence_of :name
-
+  belongs_to :restaurant
```
Comment (luciademoja) [inline app/models/menu.rb:3]: Does this solve the task about creating a relationship between restaurant and menu models? :) 
We had it as a task in an upcoming feature: https://www.pivotaltracker.com/story/show/128857113

Comment (tochman) [reply app/models/menu.rb:3]: Yes, it does. It was not practical to put this off. This story could not be delivered without this association in place.

File: features/step_definitions/steps.rb [hunk 2]
```diff
@@ -16,8 +16,13 @@
 
 
 Given(/^I am logged in as a user of the system$/) do
-  user = FactoryGirl.create(:user)
-  login_as user
+  @user = FactoryGirl.create(:user)
+  login_as @user
+end
+
+Then(/^show me an image of the page$/) do
```
Comment (luciademoja) [inline features/step_definitions/steps.rb:23]: Is this supposed to be still here?

Comment (tochman) [reply features/step_definitions/steps.rb:23]: Yes, it's a new step that renders a image of the page. We added it in this pr

**Pull request merged**